docs/rules/jsx-first-prop-new-line.md

```diff
@@ -2,13 +2,21 @@
 
 Ensure correct position of the first property.
 
+**Fixable:** This rule is automatically fixable using the `--fix` flag on the command line.
+
 ## Rule Details
 
 This rule checks whether the first property of all JSX elements is correctly placed. There are three possible configurations:
 * `always`: The first property should always be placed on a new line.
 * `never` : The first property should never be placed on a new line, e.g. should always be on the same line as the Component opening tag.
 * `multiline`: The first property should always be placed on a new line when the properties are spread over multiple lines.
 
+In order to utilise autofix, please specify the indentation style as an additional argument in your configuration:
```

Comment:
- Removed all traces of indentation argument
- `utilize`
- Added a simple warning line in readme that the fixer will require a second run of lint to fix indentation